packages/theme-chalk/src/pagination.scss

```diff
@@ -77,6 +77,9 @@
       background-color: getCssVar('pagination-button-disabled-bg-color');
       cursor: not-allowed;
     }
+    & +.#{$namespace}-pagination__sizes{
```

Comment:
## Suggest

minor opinion, but some code should be cleaned up for consistency with others.

```css
& + .#{$namespace}-pagination__sizes { // here

}
```

```css
margin-left: 16px; // here
```

also, I think the 177 to 178 line code is unnecessary.